api/client.js

```diff
@@ -182,6 +182,19 @@ function connectAndLogin(url, credentials, options, callback) {
 }
 
 
+/**
+  Returns a url that is to be used to connect to a supplied model uuid on the
```

Comment:
s/url/URL/ in this line and the following.

Reply:
done